.github/workflows/test.yml

```diff
@@ -74,19 +74,19 @@ jobs:
         sudo chown -R rootless.rootless /home/rootless
 
     - name: integration test (fs driver)
-      run: sudo -E PATH="$PATH" script -e -c /bin/bash -c 'stty rows 40 cols 80; TERM=xterm make local${{ matrix.rootless }}integration'
+      run: sudo -E PATH="$PATH" script -e -c 'make local${{ matrix.rootless }}integration'
```

Comment:
Do we still need `script`?

Reply:
In this case yes -- there's no terminal otherwise, and there' no `ssh -tt` to make one (although I was thinking about ssh-ing to locahost for the sake of using `-tt`, but `script` looks more elegant).

Also, in this case script is working fine -- I think the only issue is with the one from CentOS 7.

Reply:
Please add that to comment lines? Then LGTM

Reply:
Yep, I had this comment and then it was removed.

Added.